woocommerce/woocommerce-gutenberg-products-block
# Rename Centered Header Menu with Search pattern

## What

Rename the `Centered Header Menu with Search` to `Centered Header Menu`.

Fixes #11523

## Why



Since the search bar has been removed from this pattern, this PR renames the pattern title and slug to reflect that change.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new page or post.
2. Confirm that the `Centered Header Menu` pattern appears under the WooCommerce category dropdown and is NOT named `Centered Header Menu with Search`.
3. Insert in and confirm everything appears as expected.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
| ![CleanShot 2023-11-06 at 13 13 57](https://github.com/woocommerce/woocommerce-blocks/assets/481776/4e87e971-da25-4265-abeb-21aa4a65a323) | ![CleanShot 2023-11-06 at 12 58 16](https://github.com/woocommerce/woocommerce-blocks/assets/481776/c44daa9e-ef1e-4f1f-a75c-88acb7f7844d) |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Rename the `Centered Header Menu with Search` to `Centered Header Menu`.
